jsonschema-core.xml

```diff
@@ -904,6 +905,86 @@
                 </section>
             </section>
 
+            <section title='Recursive references with "$recurse"'>
+                <t>
+                    This keyword's value MUST be the boolean literal true.
+                    <cref>
+                        Future drafts may extend the usage with other values.  The immediate
+                        use case does not require any targets other than the entry point
+                        root, and a regular fragment URI reference does not provide the
+                        correct semantics.  Should other values be added in the future,
+                        it is expected that a boolean true value will remain an alias for
+                        this original use case.
+                    </cref>
+                </t>
+                <t>
+                    The presence of this keyword with a boolean true value indicates that,
+                    during processing, it MUST be treated as a reference to the schema document
+                    where processing was initiated.
```

Comment:
> it MUST be treated as a reference to the schema document where processing was initiated.

vs 

>  Note that even if processing began at a subschema within a document, the "$recurse" target MUST be the root of the document.

The first seems like its saying if we ref into a document at some lower level, it should recurse to THAT level, but the 2nd says it must be the root? Confusing.

Reply:
Yeah, the key here is the wording "schema document" which is defined in [section 4.3](https://tools.ietf.org/html/draft-handrews-json-schema-01#section-4.3) in both draft-07 and the current HEAD commit.  A schema document is a document, not a JSON object.  So saying that it _MUST be treated as a reference to the **schema document** where processing was initiated_ means exactly what the clarification says.

But I'm open to suggestion on how to word it better, preferably without entirely re-stating 4.3.  Maybe just an `<xref />` to it?  I suspect the complete list of people who understand the distinction without having to look it consists of is @awwright and me.